src/platforms/Gather/GatherGuestForm.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  FormControl,
+  HStack,
+  Icon,
+  Input,
+  Stack,
+  Text,
+  Tooltip,
+} from "@chakra-ui/react"
+import { AddGatherFormType } from "components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/AddGatherPanel"
+import { SectionTitle } from "components/common/Section"
+import { Question } from "phosphor-react"
+import { useFormContext } from "react-hook-form"
+
+const GatherGuestForm = () => {
+  const { register } = useFormContext<AddGatherFormType>()
+
+  return (
+    <>
+      <FormControl>
```

Comment:
We shouldn't use FormControl as a high level wrapper like this and have multiple inputs in one. We should replace the Stacks around the labels + inputs below instead to be individual FormControls